src/Compilers/Core/Portable/StrongName/StrongNameKeys.cs

```diff
@@ -95,7 +95,6 @@ internal static StrongNameKeys Create(string keyFilePath, CommonMessageProvider
 
             try
             {
-                Debug.Assert(PathUtilities.IsAbsolute(keyFilePath));
```

Comment:
This is an annoyingly nosy assert. It gets triggered all the time a relative key path is passed through cmd line or RSP file.(which otherwise works fine in Release builds)

Since it is not guarding any internal invariant, is not actionable and is completely ignorable, it feels like removing it is a good thing.